descriptors/ModuleDescriptor-template.json

```diff
@@ -4,20 +4,24 @@
   "requires": [
     {
       "id": "login",
-      "version": "7.0"
+      "version": "7.2"
```

Comment:
Are we using something in login that requires 7.2 over 7.0?

Reply:
Ah I just updated them to the latest version. Should I keep it at the old version if there is no need?

Reply:
Ah ok. I'll keep that in mind.